stdlib/array.gr

```diff
@@ -168,13 +184,16 @@ export let append = (array1, array2) => {
   let len1 = length(array1)
   let len2 = length(array2)
 
-  init(len1 + len2, n => {
-    if (n < len1) {
-      array1[n]
-    } else {
-      array2[n - len1]
-    }
-  })
+  init(
```

Comment:
This is the Reason way of formatting lambdas in function args, let me know what you think.

Reply:
Fixed, looks like the original now